### ISSUE TITLE
start_all_services.bash does not work.

### ISSUE THREAD

**Issue by williamColocho** [slaclab/simulacrum#15]
Maybe it's meant for the docker install? It just gives error messages about files not found.  I don't fully understand how is it that it should work since it uses full paths like /model_service/model_service.py

**Comment by mattgibbs** [comment]
Yep - its meant only for the docker install.  My plan is to delete this file entirely, and then get the supervisor stuff integrated into the git repo itself, in a way that will work for both docker *and* the machines at SLAC.  That way there will just be one simple way to control services.
